android/src/main/java/com/internxt/mobilesdk/services/FileSystem.kt

```diff
@@ -57,6 +57,15 @@ object FS {
   fun createFile(path: String) : Boolean{
     return File(path).createNewFile()
   }
+
+  fun getFilenameFromPath(path: String): String {
+    return path.substring(path.lastIndexOf(File.separator)+1);
+  }
+
+  fun getFileTypeFromPath(path: String): String {
+    val filename = getFilenameFromPath(path)
```

Comment:
Does exist a situation where a given file does not have an extension?

Reply:
Yep, just throw if not at the beginning, as not being able to detect that error explicitly could be a debugging pain